velox/functions/sparksql/tests/StringTest.cpp

```diff
@@ -813,5 +819,20 @@ TEST_F(StringTest, replace) {
       replace("123\u6570\u6570\u636E", "\u6570\u636E", "data"),
       "123\u6570data");
 }
+
+TEST_F(StringTest, findInSet) {
+  EXPECT_EQ(findInSet("ab", "abc,b,ab,c,def"), 3);
+  EXPECT_EQ(findInSet("abc", "abc,b,ab,c,def"), 1);
```

Comment:
Is unicode considered in the implementation? I'm assuming the spark function supports it. Let's add tests for unicode input.